README.md

```diff
@@ -15,24 +15,23 @@
   </a>
 </p>
 
-English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.zh-hk.md) | [العربية](./i18n/README.ar-dz.md) | [Italiano](./i18n/README.it-it.md) | [Українська](./i18n/README.ua-ua.md) | [Español](./i18n/README.es-es.md) | [Português](./i18n/README.pt.md) | [日本語](./i18n/README.ja-ja.md)
+English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.zh-hk.md) | [العربية](./i18n/README.ar-dz.md)
+| [Italiano](./i18n/README.it-it.md) | [Українська](./i18n/README.ua-ua.md) | [Español](./i18n/README.es-es.md)
+| [Português](./i18n/README.pt.md) | [日本語](./i18n/README.ja-ja.md)
 
 # QuestDB
 
-QuestDB is an open-source [time-series database](https://questdb.io/glossary/time-series-database/) for high throughput ingestion and
-fast SQL queries with operational simplicity. It supports schema-agnostic
-ingestion using the InfluxDB line protocol, PostgreSQL wire protocol, and a REST
-API for bulk imports and exports.
+QuestDB is an open-source [time-series database](https://questdb.io/glossary/time-series-database/) for high throughput
+ingestion and fast SQL queries with operational simplicity. It supports schema-agnostic ingestion using the InfluxDB
+line protocol, PostgreSQL wire protocol, and a REST API for bulk imports and exports.
 
-QuestDB is well suited for financial market data, application metrics, sensor
-data, real-time analytics, dashboards, and infrastructure monitoring.
+QuestDB is well suited for financial market data, application metrics, sensor data, real-time analytics, dashboards, and
+infrastructure monitoring.
 
-QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL
-extensions make it simple to correlate data from multiple sources using
-relational and time-series joins. We achieve high performance by adopting a
-column-oriented storage model, parallelized vector execution, SIMD instructions,
-and low-latency techniques. The entire codebase is built from the ground up in
-Java and C++, with no dependencies and zero garbage collection.
+QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to correlate
+data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
+column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
+codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
```

Comment:
```suggestion
QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to correlate
data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
```

This feedback is likely out of scope for this PR. However, the section is interesting information, but maybe a lower section about "QuestDB internals" would fit better. The reason is that we deliver the core benefit, then state who it is for. After that, we want people to get into "Try QuestDB" as soon as possible.

Reply:
In this PR I was just adding a link, but for sure if you feel changing the README intro makes sense go for it :). I believe this part in particular was driven mostly by Nic after discussing on Slab about the intro